.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "src/kokkos"]
     path = src/kokkos
     url = https://github.com/kokkos/kokkos.git
+[submodule "src/HighFive"]
```

Comment:
This should be made optional, we don't want to pull highfive on a systematic basis.

Reply:
I initially was using the HighFive API for HDF5 dumps but encountered some problems with parallel io and also the io times were karge due to overhead. I rewrote the entire thing to native HDF5 API and so I don't need HighFive anymore. I forgot to remove this from `.gitmodules`. Getting rid of this.